### PR TITLE
Quarkiverse extensions should use quarkiverse-parent 7 and JDK 11

### PR DESCRIPTION
**Pull request by gastaldi**

### CODE REVIEW

**Review by ia3andy — APPROVED** [review]
LGTM

**Comment by gsmet** [comment]
I backported only the parent change to 1.13.

**Comment by gsmet** [comment]
@gastaldi the tests need an update!

**Comment by ia3andy** [comment]
@gastaldi to update snapshots, you just have to run the test(s) with `-Dsnap`

File: integration-tests/maven/src/test/resources/__snapshots__/CreateExtensionMojoIT/testCreateCoreExtension/dir-tree.snapshot [hunk 1]
```diff
@@ -25,30 +14,25 @@ integration-tests/my-ext/src/main/java/org/acme/my/
...
+integration-tests/my-ext/src/test/java/org/acme/my/
+integration-tests/my-ext/src/test/java/org/acme/my/ext/
+integration-tests/my-ext/src/test/java/org/acme/my/ext/it/
+integration-tests/my-ext/src/test/java/org/acme/my/ext/it/MyExtResourceTest.java
+integration-tests/my-ext/src/test/java/org/acme/my/ext/it/NativeMyExtResourceIT.java
 integration-tests/pom.xml
-pom.xml
+bom/
+bom/application/
+bom/application/pom.xml
```
Comment (ia3andy) [inline integration-tests/maven/src/test/resources/__snapshots__/CreateExtensionMojoIT/testCreateCoreExtension/dir-tree.snapshot:29]: It seems I should add sorting to the dirtree snapshots output to avoid those unnecessary changes..

Comment (ia3andy) [reply integration-tests/maven/src/test/resources/__snapshots__/CreateExtensionMojoIT/testCreateCoreExtension/dir-tree.snapshot:29]: https://github.com/quarkusio/quarkus/pull/16859
